src/proposals/proposal.cpp

```diff
@@ -59,15 +58,52 @@ namespace hypha
 
     void Proposal::postProposeImpl(Document &proposal) {}
 
+    void Proposal::vote(const eosio::name &voter, const std::string vote, Document& proposal)
+    {
+        proposal.getContentWrapper().getOrFail(BALLOT_OPTIONS, vote, "Invalid vote");
```

Comment:
What does this do?  getOrFail returns a value but I'm not sure what BALLOT_OPTIONS is and the value isn't kept. ?

Reply:
That's the options (fail/pass now). Is stored on the document upon creation. 
See: https://github.com/hypha-dao/dao-contracts/pull/6/files#diff-e2b01f9052b8904de7a9e7848c838cd2064c181e1fc6d376169cfa95555604c8R149